.github/workflows/static-test.yml

```diff
@@ -0,0 +1,23 @@
+name: static-test
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
```

Comment:
This action does not seem to check out master, but just merges your branch with master.